client/signup/steps/woocommerce-install/transfer/index.tsx

```diff
@@ -153,19 +31,8 @@ export default function Transfer( props: WooCommerceInstallProps ): ReactElement
 			isWideLayout={ isReskinned }
 			stepContent={
 				<>
-					<div className="transfer__heading-wrapper woocommerce-install__heading-wrapper">
-						<div className="transfer__heading woocommerce-install__heading">
-							<Title>{ step }</Title>
-						</div>
-					</div>
-					<div className="transfer__content woocommerce-install__content">
-						{ error.transferFailed && 'error...' /* todo */ }
-						{ error.transferFailed && error.transferStatus }
-						<div
-							className="transfer__progress-bar"
-							style={ { '--progress': simulatedProgress } as React.CSSProperties }
-						/>
-					</div>
+					{ isAtomic && <InstallPlugins goToStep={ goToStep } /> }
+					{ ! isAtomic && <TransferSite goToStep={ goToStep } /> }
```

Comment:
It'd be great if we could combine these two flows cleanly again but I'm not sure we can, maybe using some kind of effect hook to close over the complexity?

Reply:
Latest version of this is quite a bit cleaner, at this point the separation helps simplify things.